### PR TITLE
front end components

### PR DESCRIPTION
**Pull request by ogbanana**
Added components

### CODE REVIEW

**Review by mallockey — APPROVED** [review]
Looks good.

File: package.json [hunk 1]
```diff
@@ -4,33 +4,36 @@
   "private": true,
   "scripts": {
     "build": "next build",
+    "build:css": "tailwind build tailwindcss/styles.css -o src/styles.css",
```
Comment (penspinner) [inline package.json]: Let's remove this as Tailwind already works without this.

```suggestion
```

File: src/components/EditUserProfile.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React, { useRef } from 'react'
+import Axios from 'axios'
+
+const EditUserProfile = () => {
+  const user = JSON.parse(window.localStorage.getItem('cookie'))
```
Comment (penspinner) [inline src/components/EditUserProfile.tsx]: Using `window` during render will cause Next.js build to break. Can store a null value in state and then set the user in an effect.

File: src/pages/UserProfile.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+import Axios from 'axios'
+import Header from '../components/Header'
+import Footer from '../components/Footer'
+import EditUserProfile from '../components/EditUserProfile'
+import OrderHistoryItem from '../components/OrderHistoryItem'
+
+const UserProfile = () => {
+  const [cookie, setCookie] = useState(null)
+  const [localStorage, setLocalStorage] = useState(null)
+  const user = JSON.parse(window.localStorage.getItem('cookie'))
```
Comment (penspinner) [inline src/pages/UserProfile.tsx]: Using `window` on render.

File: src/utils/productDummyData.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
+module.exports = [
```
Comment (penspinner) [inline src/utils/productDummyData.js:1]: Use `export default`.

**Review by penspinner — CHANGES_REQUESTED** [review]
Remove `src/styles.css` since it's not needed in Git.

File: src/components/EditUserProfile.tsx (hunk 2 above)
Comment (ogbanana) [reply src/components/EditUserProfile.tsx]: k

File: quickcommit.sh [hunk 5]
```diff
@@ -0,0 +1,3 @@
+git add .
+git commit -m $1
+git push origin jPlateComponents
```
Comment (penspinner) [inline quickcommit.sh]: Let's remove this file as well. This should be an alias/utility available on your machine rather than on the GitHub repo.

**Review by penspinner — CHANGES_REQUESTED** [review]

File: src/utils/productDummyData.js (hunk 4 above)
Comment (ogbanana) [reply src/utils/productDummyData.js:1]: k

**Review by penspinner — APPROVED** [review]

**Pull request merged**